crates/fluvio-compression/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "fluvio-compression"
+version = "0.0.0"
```

Comment:
Since this is going to be depended on by public create, it should have non zero version

Reply:
done